examples/webgl_buffergeometry_points_glbufferattribute.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - buffergeometry [particles]</title>
```

Comment:
Maybe renaming the title to a more descriptive name: `three.js webgl - buffergeometry - custom VBOs`.

Reply:
Ok, see bbc5638

Reply:
Done